src/lib/wsman-filter.c

```diff
@@ -161,7 +161,7 @@ filter_t * filter_create_selector(hash_t *selectors)
 static int
 _filter_add_selector(filter_t *filter, const char* key, const char *value, const epr_t *epr)
 {
-	int i;
+	unsigned int i;
```

Comment:
while you're at it, please move the declaration of `i` into the `if (filter_cpy->selectorset.selectors) {` block. Just like it's done for the `if(filter->resultProp) {` block in the same function.